cis_proc.py

```diff
@@ -254,10 +259,10 @@ def main(tar_file, bids_dir, config, sub, ses=None, work_dir=None, n_procs=1):
             os.makedirs(op.join(out_deriv_dir, 'logs'))
 
         # Run MRIQC anat
-        for tmp_mod in config_options['mriqc_options']['anat']['mod'].keys():
+        mriqc_anat_modality = config_options['mriqc_options']['anat']['mod']
```

Comment:
I know this is going to get refactored soon-ish, but I assigned the config options for `{'anat': {'mod': []}}` and  `{'func': {'task': []}}` names to increase readability